Chocolatey.Cake.Recipe/Content/credentials.cake

```diff
@@ -52,6 +52,25 @@ public class PackageSourceCredentials
     }
 }
 
+public class DockerSourceCredentials
```

Comment:
Can you rename this to just `DockerCredentials`?  I don't think `Source` adds anything here.

Reply:
Done!